stockanalyser/analysis/levermann.py

```diff
@@ -190,16 +190,14 @@ def __init__(self, stock):
         self.stock = stock
         self.evaluation_results = []
 
-        if ".de" not in self.stock.symbol.lower():
-            raise NotSupportedError("Only DAX Stocks are supported."
-                                    " The stock symbol has to end in .de")
-
         if self.stock.cap_type == Cap.LARGE:
             self.reference_index = "^GDAXI"
         elif self.stock.cap_type == Cap.MID:
             self.reference_index = "^MDAXI"
         elif self.stock.cap_type == Cap.SMALL:
             self.reference_index = "^SDAXI"
+        else:
+            self.reference_index = "^UNKNOWN"
```

Comment:
What was the idea behind setting it to "^UNKNOWN"?
Fetching quotes for a reference index called ^UNKNOWN should fail.
Therefore I think it's better to keep the previous approach, throwing an exception if we don't know the reference index